third_party/mesa/src/util/simple_mtx.h

```diff
@@ -110,10 +114,18 @@ simple_mtx_lock(simple_mtx_t *mtx)
    if (__builtin_expect(c != 0, 0)) {
       if (c != 2)
          c = p_atomic_xchg(&mtx->val, 2);
-      while (c != 0) {
-         futex_wait(&mtx->val, 2, NULL);
-         c = p_atomic_xchg(&mtx->val, 2);
-      }
+// GOOGLE: Windows does not use futexes.
+#ifdef _WIN32
+        while (c != 0) {
+            WaitForSingleObject(mtx->val, 2);
```

Comment:
can we define `futex_wait` in google_dive_ext.h/c as well? (with the actual implementation with `WaitForSingleObject`)